routes/users/login.js

```diff
@@ -10,15 +10,26 @@ const login = (req, res) => {
   const pw = md5(password)
   MongoClient.connect(URI, { useNewUrlParser: true }, (err, db) => {
     if (err) throw err
-
     db.db(DB_NAME)
       .collection("korisnici")
       .findOne({ email, pass: pw }, (err, user) => {
         if (!user) {
           return res.status(400).send(new ErrRes("Pogresan email ili password"))
         }
-        const token = jwt.sign({ user }, tokenKey)
-        res.send(new SuccRes("Token", token))
+        const token = jwt.sign({ user }, tokenKey, { expiresIn: "30d" })
+        res.send(new SuccRes("Token sent!", token))
+        const tokenModel = {
+          userId: user._id,
+          token,
+          dodat: user._id.getTimestamp()
```

Comment:
mislim da se potkrala greska, ovde pise da je token dodat u vreme kada je registrovan korisnik, sto moze biti godinama ranije.

Reply:
u principu ti ne treba to polje, duplira izvor istine, a to uvek vodi bagovima

Reply:
stavio sam u varijablu Date(), pa u objekat upisujem tu varijablu.toString()